content/2024/_index.md

```diff
@@ -12,3 +12,9 @@ There's a list of [people attending]({{< ref "people" >}}) here. You might know
 
 ## What?
 There's various [services]({{< ref "services" >}}) and [things]({{< ref "things" >}}) present that might interest you, if you choose to visit our little village.
+
+## Where?
+We'll be up in the North East Corner somwhere.
```

Comment:
> somwhere

Speeling

Reply:
Is it fixed now?  I did commit something